subworkflows/nf-core/bam_qc_picard/main.nf

```diff
@@ -0,0 +1,43 @@
+//
+// Run QC steps on BAM/CRAM files using Picard
+//
+
+params.options = [:]
+
+include { PICARD_COLLECTMULTIPLEMETRICS } from '../../../modules/picardcollectmultiplemetrics/main' addParams( options: params.options )
+include { PICARD_COLLECTWGSMETRICS      } from '../../../modules/picardcollectwgsmetrics/main'      addParams( options: params.options )
+include { PICARD_COLLECTHSMETRICS       } from '../../../modules/picardcollecthsmetrics/main'       addParams( options: params.options )
+
+workflow BAM_QC_PICARD {
+    take:
+    ch_bam_bai          // channel: [ val(meta), [ bam ], [bai/csi] ]
+    ch_fasta            // channel: [ fasta ]
+    ch_bait_interval    // channel: [ bait_interval ]
+    ch_target_interval  // channel: [ target_interval ]
+
+    main:
+    ch_versions = Channel.empty()
+
+    PICARD_COLLECTMULTIPLEMETRICS( ch_bam_bai, ch_fasta] )
+    ch_versions = ch_versions.mix(PICARD_COLLECTMULTIPLEMETRICS.out.versions.first())
+    if (!ch_bait_interval.isEmpty() || !ch_target_interval.isEmpty()) {
+        if (ch_bait_interval.isEmpty()) {
+            throw new Error("Bait interval channel is empty")
```

Comment:
why not `log.error`?

Reply:
This was a suggestion by github copilot, I'll fix it!